extensions/amp-embedly-card/1.0/storybook/Basic.amp.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '#preact';
+import {withAmp} from '@ampproject/storybook-addon';
+import {withKnobs} from '@storybook/addon-knobs';
+
+export default {
+  title: 'amp-embedly-card-1_0',
+  decorators: [withKnobs, withAmp],
+
+  parameters: {
+    extensions: [{name: 'amp-embedly-card', version: '1.0'}],
+    experiments: ['bento'],
+  },
+};
+
+export const YoutubeEmbed = () => {
+  return (
+    <>
+      <amp-embedly-key layout="nodisplay" value="my-test-key">
+        hello
```

Comment:
This can be removed since it is `layout="nodisplay"`.
```suggestion
```

Reply:
Sorry for the mistake. It was for the test purpose and somehow I forgot to remove it 🤔